compiler-core/src/error/snapshots/gleam_core__error__tests__bun-linux-ubuntu.snap

```diff
@@ -0,0 +1,8 @@
+---
+source: compiler-core/src/error/tests.rs
+expression: "err[0].text"
+---
+The program `bun` was not found. Is it installed?
```

Comment:
An empty line between this line and the line sharing the docs please :pray:

Reply:
ok